### DISCUSSION TITLE
How translate boolean values in filter?

### DISCUSSION THREAD

**Discussion by risentveber** [activeadmin/activeadmin#8060] [Help] [answered]
```ruby
filter :completed
```
results in 
<img width="266" alt="image" src="https://github.com/activeadmin/activeadmin/assets/6682918/c894bea9-4f39-424d-8360-daca59c3ac5d">
```yaml
yes: Да
no: Нет
```

I couldn't find any I18t key to mitigate the issue.

**Comment by javierjulio** [comment, marked as answer]
@risentveber since this is for filters which relies on Formtastic, I believe its coming from there. [Try overriding the yes/no keys](https://github.com/formtastic/formtastic_i18n/blob/master/config/locales/en.yml) and let us know if that works. Thanks.

**Comment by budu** [comment]
I've had a similar issue. When the attribute is a boolean, you'll want to add translations for the `:true`/`:false` keys, for example:

```yaml
fr:
  formtastic:
    :true: "Oui"
    :false: "Non"
```
